src/client/service.h

```diff
@@ -178,35 +188,43 @@ class TCPIPAcceptor : public I2PService {
   virtual ~TCPIPAcceptor() {
     TCPIPAcceptor::Stop();
   }
-
-  // If you override this make sure you call it from the children
+  /// @brief Start the handler; If you override this make sure you call it from
+  /// the children
   void Start();
-
-  // If you override this make sure you call it from the children
+  /// @brief stop the handler;  If you override this make sure you call it from
+  /// the children
   void Stop();
-
-  // stop tunnel, change address, start tunnel
-  // will throw exception if the address is already in use
+  /// @brief stop tunnel, change address, start tunnel
+  /// will throw exception if the address is already in use
   void Rebind(
       const std::string& addr,
       std::uint16_t port);
-
+  /// @brief get the endpoint
   // @return the endpoint this TCPIPAcceptor is bound on
   boost::asio::ip::tcp::endpoint GetEndpoint() const {
     return m_Acceptor.local_endpoint();
   }
 
  protected:
+  /// @brief  create handler object
+  /// @param  socket pointer to transfer
+  /// @return return a shared pointer to the base class of this handler;
   virtual std::shared_ptr<I2PServiceHandler> CreateHandler(
       std::shared_ptr<boost::asio::ip::tcp::socket> socket) = 0;
-
+  /// @brief get name of service
+  /// @return std::string name of service
   std::string GetName() const { return "generic TCP/IP accepting daemon"; }
 
  protected:
   std::string m_Address;
 
  private:
+  /// @brief accept connection ; create socket for handler to listen on
+  /// pass handler function; this is what starts the communication
   void Accept();
+  /// @brief callback function to handle data transfers ;
+  /// @param ecode
```

Comment:
I could've sworn I put this in the last PR but it apparently didn't go through.

You should put something like `boost error code` instead of leaving blank.